src/pages/AssignServerRoles/ModelServerDetails.js

```diff
@@ -45,6 +45,8 @@ class ModelServerDetails extends Component {
           {this.renderTextLine('server.name.prompt', this.props.data.name)}
           {this.renderTextLine('server.role.prompt', this.props.data.role)}
           {this.renderTextLine('server.ip.prompt', this.props.data['ip-addr'])}
+          {this.renderTextLine('server.ip.prompt', this.props.data['release'])}
```

Comment:
This page shows parameters that have been manually entered, i.e. not the fields that were discovered when talking to SUMA or oneview. There is no place in the UI where these are manually entered, so they should not appear here. Note also that they use they wrong label here (3 fields on the page are now labelled as IP address)